packages/cyberstorm/src/components/Switch.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { PropsWithChildren, useCallback } from "react";
+import styles from "./componentStyles/Switch.module.css";
+import * as RadixSwitch from "@radix-ui/react-switch";
+
+export interface SwitchProps {
+  state: boolean;
+  switchCallback?: (state: boolean) => void;
```

Comment:
Related to this earlier comment:
> Seems like a rather pointless wrapper. Why not just use setState as the prop?

the callback could then use typings imported from `React`: `Dispatch<SetStateAction<boolean>>`. Unless there's some idea that we want the toggle to do something else than actually toggle a state 🤷🏻

Reply:
There was no idea 😅 I've updated the typing 👍